src/common/ini.c

```diff
@@ -358,27 +358,23 @@ iniLoad(
 
             do
```

Comment:
Not sure where to leave this comment - but I am glad to see that the reading of pgbackrest.conf (and thus iniParse) is not affected since I have seen people put comments in the conf file. However, I might suggest in the ini.h file that it is made clear that iniLoad should never be used for the conf files - only iniParse.

Reply:
Yeah, these are separate code paths and now they really need to stay that way. Added some comments to help clarify what to use for what.

I added support for # and [ since I was in there and it was clear that they would also not work as expected. Trying to save us an issue down the road.